pos-for-customer/order/pos-customer-order.page.html

```diff
@@ -197,9 +202,9 @@ <h1>{{ this.id ? 'ĐÃ CẬP NHẬT ĐƠN HÀNG' : 'ĐÃ TIẾP NHẬN ĐƠN HÀ
 			</ion-modal>
 
 
-			<ion-button *ngIf="item.Status =='New' || item.Status =='Done'" class="ion-margin-top" expand="block" (click)="processPayments()" [disabled]="item._TotalQuantity <= 0 || !pageConfig.canEdit">
-				<ion-text *ngIf="item.Status =='New'">Thanh toán</ion-text>
-				<ion-text *ngIf="item.Status =='Done'">Xem chi tiết</ion-text>
+			<ion-button *ngIf="item.Status != 'Cancelled'" class="ion-margin-top" expand="block" (click)="processPayments()">
```

Comment:
Các đơn đã chia, đã gộp nữa nha.
Vì khi chia hoặc gộp thì đơn gốc chỉ để đối chiếu chứ không sử dụng tính toán.